Fix `listing` block in SSR, now it is fully variations aware, and the… (#3684)
plone/volto: CHANGELOG.md

### Bugfix

> - Fix `listing` block in SSR, now it's fully variations aware, and the config is passed to the SSR querystring action @sneridagh

Reviewer: ```suggestion
- Fix `listing` block in SSR, now that it is fully variations aware and the configuration is passed to the SSR `querystring` action. @sneridagh
```
